mscraftsman/devcon2020
# WIP: Feature/sponsors and stats (and speakers and sessions)

Work In Progress: Please do **NOT** merge yet.

DONE:
- add the 2020 links for the sheets & sessionize

TODO:
- remove demo code, once dev validated

How to test
1. Check the logs in the console showing the data for each entity.
2. Check that everything looks good in the Store.

**NB**: the data for speakers & sessions is not yet available for 2020.
